### DISCUSSION TITLE
Where does *:not(style) ~ *:not(style) and how to override?

### DISCUSSION THREAD

**Discussion by hutber** [chakra-ui/chakra-ui#2469] [Q&A] [answered]
Currently I am having to override most of my useBreakpointvalue items with an important, but this means I cannot use my number based fontSizes, lineHeights etc.

  const margin = useBreakpointValue({ base: "0 !important", md: space["0.5"] });
So in this situation I cannot use `important` on my `space`

What is this style and how can I find out where it is defined?

**Comment by chasinhues** [comment, marked as answer]
hey @hutber , can you please provide a bit more context on what you're trying to accomplish? It might be that there is a better approach to what you're trying to do, but I can't be sure without more context 😄
